tests/Controllers/SalesControllerTest.php

```
@@ -0,0 +1,265 @@
+        // "sales_stock" is also required: Employee::has_module_grant('sales', ...)
+        // treats the bare "sales" grant as insufficient once any sales_* submodule
+        // permission exists in the permissions table (see has_subpermissions()).
+        $grantsData = [
+            ['permission_id' => 'sales', 'menu_group' => 'home'],
+            ['permission_id' => 'sales_stock', 'menu_group' => 'home']
+        ];
+
+        $employeeModel = model(Employee::class);
+        $employeeModel->save_employee($personData, $employeeData, $grantsData, NEW_ENTRY);
+
+        return (int) $personData['person_id'];
```

PHPStan (2.2.7)

[error] Offset 'person_id' does not exist on array{first_name: 'Cashier', last_name: 'NoReports', email: non-falsy-string, phone_number: '555-0001'}.


(offsetAccess.notFound)